__tests__/app.test.js

```diff
@@ -113,10 +113,77 @@ describe('GET /api/topics', () => {
               votes: expect.any(Number),
               article_img_url: expect.any(String),
               comment_count: expect.any(String)
-            });
-          });
-        });
-    });
+            })
+          })
+        })
+    })
     
-  });
+  })
+
+  describe("GET /api/articles/:article_id/comments", () => {
+    test('200 OK: empty array when article_id is valid but has no comments', () => {
+      return request(app)
+        .get('/api/articles/2/comments')
+        .expect(200)
+        .then((response) => {
+          expect(response.body.comments).toEqual(expect.any(Array));
+        });
+        })
+
+    test("200 OK: all comments for an article", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body
+          expect(comments).toBeInstanceOf(Array)
+          expect(comments).toHaveLength(11) 
+          
+          comments.forEach((comment) => {
+            expect(comment).toHaveProperty("comment_id")
+            expect(comment).toHaveProperty("body")
+            expect(comment).toHaveProperty("votes")
+            expect(comment).toHaveProperty("author")
+            expect(comment).toHaveProperty("article_id")
+            expect(comment).toHaveProperty("created_at")
 
+            expect(comment).toMatchObject({
```

Comment:
If we have this block here, it makes the previous block checking for the keys redundant.